src/sentry/logging/handlers.py

```diff
@@ -20,8 +20,11 @@ def emit(self, record):
         if record.exc_info:
             kwargs['exc_info'] = record.exc_info
 
-        if isinstance(record.args, dict):
-            kwargs.update(record.args)
+        if record.args:
+            if isinstance(record.args, dict):
+                kwargs.update(record.args)
```

Comment:
Are we still trying to avoid having a nested `args` dict?

Otherwise, why not just do:

`kwargs['args'] = record.args` and let jesus take the wheel.